Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -184,6 +184,22 @@ theorem Subset.congr_right : ∀ {x y z : PSet}, Equiv x y → (z ⊆ x ↔ z 
       let ⟨a, ab⟩ := βα b
       ⟨a, cb.trans (Equiv.symm ab)⟩⟩
 
+instance : Preorder PSet where
+  le := (· ⊆ ·)
+  le_refl := refl_of (· ⊆ ·)
+  le_trans _ _ _ := trans_of (· ⊆ ·)
+
+instance : HasSSubset PSet :=
+  ⟨(· < ·)⟩
```

Comment:
I'd put this on one line, and maybe similar for the below

Reply:
Not personally a fan of having attributes on one-line theorems, so I'll keep those as is.